README.md

```diff
@@ -234,6 +234,23 @@ If you aren’t using React, you can still render a player using the standalone
 
 See [`jsFiddle` example](https://jsfiddle.net/krkcvx9s/)
 
+#### Adding your own players
+
+If you have your own player, that is compatible with ReactPlayer's internal architecture, you can use it like this:
+
+```javascript
+import YourOwnPlayer from './somewhere';
+ReactPlayer.addAdditionalPlayer(YourOwnPlayer);
+```
+
+Or you can clear all additional players:
+
+```javascript
+ReactPlayer.clearAdditionalPlayers();
```

Comment:
Slightly anal, but I think I prefer referring to the extra players as "custom players" rather than "additional players" throughout the codebase.

Reply:
@CookPete so then `ReactPlayer.addCustomPlayer()` and `ReactPlayer.clearCustomPlayers()`?

Reply:
`addCustomPlayer` and `removeCustomPlayers` would be better I think 👍